src/main/java/net/atos/zac/app/zaken/ZakenRESTService.java

```diff
@@ -899,6 +899,7 @@ private void koppelHoofdEnDeelzaak(final Zaak hoofdzaak, final UUID deelzaakUUID
         System.out.println("koppelHoofdEnDeelzaak ZAAK.updated " + hoofdzaak.getUuid());
         // Hiervoor wordt door open zaak alleen voor de deelzaak een notificatie verstuurd.
         // Dus zelf het ScreenEvent versturen voor de hoofdzaak!
+        indexeerService.addZaak(hoofdzaak.getUuid(), true);
```

Comment:
Als alleen de zaak zelf is gewijzigd hoeven niet ook de taken te worden geindexeerd.